exercises/allergies/test/Test.hx

```diff
@@ -0,0 +1,291 @@
+package;
+
+import Allergies.Allergen;
+
+using buddy.Should;
+
+class Test extends buddy.SingleSuite {
+	public function new() {
+		describe("testing for eggs allergy", {
+			it("not allergic to anything", {
+				var sut = new Allergies(0);
+				sut.allergicTo(Allergen.Eggs).should.be(false);
+			});
+			it("allergic only to eggs", {
+				pending("Skipping");
+				var sut = new Allergies(1);
+				sut.allergicTo(Allergen.Eggs).should.be(true);
+			});
+			it("allergic to eggs and something else", {
+				pending("Skipping");
+				var sut = new Allergies(3);
+				sut.allergicTo(Allergen.Eggs).should.be(true);
+			});
+			it("allergic to something, but not eggs", {
+				pending("Skipping");
+				var sut = new Allergies(2);
+				sut.allergicTo(Allergen.Eggs).should.be(false);
+			});
+			it("allergic to everything", {
+				pending("Skipping");
+				var sut = new Allergies(255);
+				sut.allergicTo(Allergen.Eggs).should.be(true);
+			});
+		});
+		describe("testing for peanuts allergy", {
+			it("testing for peanuts allergy", {
+				pending("Skipping");
```

Comment:
Each story, a describe statement in this case, should have its first test case run.  So you can remove the `pending("skipping");` from the first `it()` under every describe.